roles/backend_setup/tasks/vg_create.yml

```diff
@@ -41,7 +41,7 @@
 
 - name: Set the PE size
   set_fact:
-     vg_pesize: "{{ vg_pe | int }}"
+     vg_pesize: "{{ vg_pe | int + 0.5 | round | int }}"
```

Comment:
@daelmaselli this unconditionally bumps the number to +1
For example if vg_pe is 2, this sets the value to 3. I didn't understand the fix, can you please explain?